### PR TITLE
Fix issue #374

### PR DESCRIPTION
**Pull request by dvandyk**
@DomLeljak this commit fixed the problem you encountered, in all likelihood. Let's discuss please.

### CODE REVIEW

File: eos/utils/concrete-cacheable-observable.hh [hunk 1]
```diff
@@ -311,6 +312,15 @@ namespace eos
 
             virtual ObservablePtr make(const Parameters & parameters, const Kinematics & kinematics, const Options & options) const
             {
+                for (const auto & fo : _forced_options)
+                {
+                    const auto & key = std::get<0>(fo);
+                    if (options.has(key))
+                    {
+                        Log::instance()->message("[ConcreteObservableEntry.make]", ll_error)
```
Comment (mreboud) [inline eos/utils/concrete-cacheable-observable.hh:320]: Shouldn't this rather be `[ConcreteCacheableObservableEntry.make]`?

File: eos/utils/expression-observable.cc [hunk 2]
```diff
@@ -148,6 +149,16 @@ namespace eos
             throw InternalError("Empty expression encountered in ExpressionObservableEntry::make!");
         }
 
+        for (const auto & fo : _forced_options)
+        {
+            const auto & key = std::get<0>(fo);
+            if (options.has(key))
+            {
+                Log::instance()->message("[ConcreteObservableEntry.make]", ll_error)
```
Comment (mreboud) [inline eos/utils/expression-observable.cc:157]: Again I would rather print `[ExpressionObservableEntry.make]`

File: eos/utils/concrete-cacheable-observable.hh (hunk 1 above)
Comment (dvandyk) [reply eos/utils/concrete-cacheable-observable.hh:320]: Indeed.

File: eos/utils/expression-observable.cc (hunk 2 above)
Comment (dvandyk) [reply eos/utils/expression-observable.cc:157]: Yes

**Pull request merged**